### DISCUSSION TITLE
Where do you see the advantages of using Snappy vs Nextcloud mail App?

### DISCUSSION THREAD

**Discussion by witzker** [the-djmaze/snappymail#1127] [General]
Any experience?

**Comment by cm-schl** [comment]
Hi, there is a bit older comparison table in this issue: https://github.com/the-djmaze/snappymail/issues/569
Nextcloud Mail made some important steps improving many points - one thing that maybe is a showstopper for a production use of Nextcloud Mail could be the search functionality: https://github.com/nextcloud/mail/issues/5611
